### ISSUE TITLE
Millis timestamp format used by JS on hybrid SDKs causes logs to be flooded

### ISSUE THREAD

**Issue by jennmueng** [getsentry/sentry-java#1513]
As the JavaScript SDK uses unix timestamp in the format of seconds with millis as decimals on events, envelopes passed through the Android SDK log the error:
```
Error when deserializing UTC timestamp format, it might be millis timestamp format.
    java.lang.IllegalArgumentException: timestamp is not ISO format 1623051511.006
```
This is logged for every timestamp in the event, including breadcrumbs, so when a given event is sent tens if not hundreds of these logs show up in logcat, flooding the feed:

<img width="1071" alt="Screen Shot 2021-06-07 at 2 57 00 PM" src="https://user-images.githubusercontent.com/30991498/120980267-a614ae00-c7a0-11eb-9886-cf6503ed30fe.png">

Although the log is leveled `DEBUG`, as we are trying to have our users enable debug mode by default, this would flood their logs.
https://github.com/getsentry/sentry-java/blob/b45944c93d53f68bc1e64f5ac4c9a5400b252d44/sentry/src/main/java/io/sentry/adapters/DateDeserializerAdapter.java#L31-L40

We should consider removing this log.

**Comment by bruno-garcia** [comment]
If parsing fails on all cases. Please log an error message.

**Comment by romtsn** [comment]
This is closed by #2968
